source/time-series-collections.txt

```diff
@@ -0,0 +1,66 @@
+.. _vsce-time-series:
+
+==============================
+Create Time Series Collections
+==============================
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/fact-vsce-preview.rst
+
+You can create time series collections using the MongoDB Playground. 
+
+Prerequisites
+-------------
+
+If you have not done so already, you must complete the following 
+prerequisites before you can create documents with a MongoDB Playground:
+
+- :ref:`Create a connection to a MongoDB deployment 
+  <vsce-connect-task>`.
+- :ref:`Activate the connection to the MongoDB deployment 
+  <vsce-activate-connection>`.
+
+Create a time series collection
+-------------------------------
+
+Once you connect your deployment using MongoDB for VS code, use the left 
+navigation to: 
+
+1. Expand an active connection and hover over the database where you want 
+   your collection to exist.
+
+#. Click the :icon-fa5:`plus` icon that appears. 
+
+#. A MongoDB playground automatically opens with a template form to create 
+   both regular collections and time series collections.
+
+#. Save the playground as a ``.mongodb`` file.
```

Comment:
You don't have to save a playground before running it. Saving playgrounds is a separate feature that is available to users and it isn't a requirement for creating time-series collections.

Reply:
Done.